ui/core/src/tokenwhitelist.sandpit.json

```diff
@@ -0,0 +1,387 @@
+[
```

Comment:
What's this file?

Reply:
It's the actual whitelist stored in the smart contracts.  I figured since we're checking in everything else, this should be checked in also.

Reply:
I see